gratipay/models/participant/email.py

```diff
@@ -106,6 +103,16 @@ def validate_email_verification_request(self, c, email, *packages):
         if not all(email in p.emails for p in packages):
             raise EmailNotOnFile()
 
+        unverified_email = c.one("""
+            SELECT address
```

Comment:
For example, github: 

![screen shot 2017-08-23 at 7 49 26 pm](https://user-images.githubusercontent.com/3893573/29620987-19cdccf2-883d-11e7-92d4-7d1b751b7c04.png)